### ISSUE TITLE
Bug: cannot use two backslashes at the end of label string

### ISSUE THREAD

**Issue by vsile** [prometheus-community/json_exporter#66]
If I use two backslashes in JSON-file at the end of label string, then JSON exporter exposes only metrics before this metric.

```
{
  "metrics": [
    {
        "path": "dpc\\root\\GRs\\PK\\gr047_mirror",
        "value": "0"
    },
    {
        "path": "dpc\\root\\GR\\PK\\",
        "value": "1"
    },
    {
        "path": "dpc\\root\\GRs\\PK\\gr048_mirror",
        "value": "1"
    }
  ]
}
```
Expected:
```
# HELP zabbix_metrics_value zabbix_metrics
# TYPE zabbix_metrics_value untyped
zabbix_metrics_value{path="dpc\\root\\GRs\\PK\\gr047_mirror"} 0
zabbix_metrics_value{path="dpc\\root\\GR\\PK\\"} 1
zabbix_metrics_value{path="dpc\\root\\GRs\\PK\\gr048_mirror"} 1
```

Result:
```
# HELP zabbix_metrics_value zabbix_metrics
# TYPE zabbix_metrics_value untyped
zabbix_metrics_value{path="dpc\\root\\GRs\\PK\\gr047_mirror"} 0
```

**Comment by vsile** [comment]
The issue originates from [github.com/kawamuray/jsonpath](https://github.com/kawamuray/jsonpath) package.

I tested this file via jsonpath CLI 
```
$ ./jsonpath --file=test_export.json --path='$.metrics[*].path+' --keys
```
and got the next error:
```
"metrics"	0	"path"	"dpc\\root\\GRs\\PK\\gr047_mirror"
Unexpected character after json value token: U+0076 'v' at byte index 157
```

**Comment by vsile** [comment]
I create a PR to fix this issue
https://github.com/kawamuray/jsonpath/pull/1/files

**Comment by rustycl0ck** [comment]
This should be fixed in version `0.3.0` since the JSONPath library was migrated to a newer and maintained one. Please re-open this issue if the problem still persists.
